zbar_testing/1_Testing_barcode_images/zbar_test.py

```diff
@@ -0,0 +1,46 @@
+# Test for zbar-py
```

Comment:
Actually when I first start testing zbar-py (how it works), I didnot know that time that all my testing will become examples.